### PR TITLE
[New Rule] Exchange Mailbox via PowerShell

### PR DESCRIPTION
**Pull request by Samirbous**
New rule similar to this [one](https://github.com/elastic/detection-rules/blob/main/rules/windows/collection_email_powershell_exchange_mailbox.toml) but using Windows powershell logs 4104.

### CODE REVIEW

File: rules/windows/collection_mailbox_export_winlog.toml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+Identifies the use of the Exchange PowerShell cmdlet, New-MailBoxExportRequest, to export the contents of a primary mailbox or archive to a .pst file. Adversaries may target user email to collect sensitive information.
+"""
+false_positives = ["Legitimate exchange system administration activity."]
+from = "now-9m"
+index = ["winlogbeat-*", "logs-windows.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Exchange Mailbox Export via PowerShell"
+note = """## Triage and analysis
+### Investigating Exporting Exchange Mailbox via PowerShell
```
Comment (w0rk3r) [inline rules/windows/collection_mailbox_export_winlog.toml:21]:
```suggestion

### Investigating Exchange Mailbox Export via PowerShell

```

File: rules/windows/collection_mailbox_export_winlog.toml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+The `New-MailBoxExportRequest` cmdlet is used to begin the process of exporting contents of a primary mailbox or archive to a .pst file. Note that this is done on a per-mailbox basis and this cmdlet is available only in on-premises Exchange.
+Attackers can abuse this functionality in preparation for exfiltrating contents, which is likely to contain sensitive and strategic data.
+#### Possible investigation steps
```
Comment (w0rk3r) [inline rules/windows/collection_mailbox_export_winlog.toml:24]:
```suggestion

#### Possible investigation steps

```

File: rules/windows/collection_mailbox_export_winlog.toml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  - Check if this operation was approved and performed according to the organization's change management policy.
+  - Retrieve the operation status and use the `Get-MailboxExportRequest` cmdlet to review previous requests.
+  - By default, no group in Exchange has the privilege to import or export mailboxes. Investigate administrators that assigned the "Mailbox Import Export" privilege for abnormal activity.
+- Investigate if there is a significant quantity of export requests in the alert timeframe. This operation is done on a per-mailbox basis and can be part of a mass export.
+- If the operation was completed successfully:
+  - Check if the file is on the path specified in the command.
+  - Investigate if the file was compressed, archived, or retrieved by the attacker for exfiltration.
+### False positive analysis
+- This mechanism can be used legitimately. Analysts can dismiss the alert if the administrator is aware of the activity and it is done with proper approval.
+### Response and remediation
```
Comment (w0rk3r) [inline rules/windows/collection_mailbox_export_winlog.toml:39]:
```suggestion
  - Investigate if the file was compressed, archived, or retrieved by the attacker for exfiltration.

### False positive analysis

- This mechanism can be used legitimately. Analysts can dismiss the alert if the administrator is aware of the activity and it is done with proper approval.

### Response and remediation

```

File: rules/windows/collection_mailbox_export_winlog.toml [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+"""
+references = [
+    "https://www.volexity.com/blog/2020/12/14/dark-halo-leverages-solarwinds-compromise-to-breach-organizations/",
+    "https://docs.microsoft.com/en-us/powershell/module/exchange/new-mailboxexportrequest?view=exchange-ps",
+    "https://www.elastic.co/security-labs/siestagraph-new-implant-uncovered-in-asean-member-foreign-ministry",
+]
+risk_score = 47
+rule_id = "54a81f68-5f2a-421e-8eed-f888278bb712"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Collection", "Investigation Guide", "Elastic Endgame"]
```
Comment (w0rk3r) [inline rules/windows/collection_mailbox_export_winlog.toml:58]:
```suggestion
tags = ["Elastic", "Host", "Windows", "Threat Detection", "Collection", "Investigation Guide", "PowerShell"]
```

**Review by w0rk3r — APPROVED** [review]
🧙🏼

**Review by terrancedejesus — APPROVED** [review]
LGTM after the changes are made from the suggestions. Great work!

**Pull request merged**